src/frontend/js/hooks/useUnionResource/index.ts

```diff
@@ -94,7 +94,11 @@ const useUnionResource = <
 
   // to force execution of useEffect::fetchNewPage(),
   // reset need to generate a uniq key that is part of it's dependencies.
-  const reset = () => {
+  const reset = (eofKey?: string) => {
```

Comment:
What I suggest is to not have a param passed to the `reset` function, to make sure it stay simple. And to reset both list each time an invalidation occur, WDYT? After all we can consider that if there is one invalidation we should reset the entire list as they nearly the same entity

Reply:
Ok deal if it works well